### PR TITLE
Update Zowe PI & Sprint Cadence.md

### PR DESCRIPTION
**Pull request by balhar-jakub**
Change the dates for 3.0.1 and 3.1.0

### CODE REVIEW

File: Project Management/Schedule/Zowe PI & Sprint Cadence.md [hunk 1]
```diff
@@ -1,123 +1,119 @@
...
 
-#### 3.0 </br>
-- System Demo 2024/10/06<br>
-- GA 2024/09/30<br>
+#### 3.0 
+- System Demo 2024, October 06
+- GA 2024, September 27
+
+#### 3.1.0 
+- Code Freeze: January 02, 2024
```
Comment (adam-wolfe) [inline Project Management/Schedule/Zowe PI & Sprint Cadence.md:24]: The year should be 2025

Comment (balhar-jakub) [reply Project Management/Schedule/Zowe PI & Sprint Cadence.md]: Thanks for catching this.

File: Project Management/Schedule/Zowe PI & Sprint Cadence.md [hunk 2]
```diff
@@ -1,123 +1,119 @@
...
-#### 1.28.7
-- Code Freeze 2024/05/14<br>
-- RC Build 2024/05/15<br>
-- Testing 2024/05/15 - 2024/05/26<br>
-- GA 2024/05/27<br>
+#### 1.28.8
+- Code Freeze: October 1, 2024
+- RC Build: October 2, 2024
+- Testing: October 2, 2024 - October 14, 2024
+- GA: October 14, 2024
```
Comment (zFernand0) [inline Project Management/Schedule/Zowe PI & Sprint Cadence.md:117]: Final V1 release 👀

Comment (balhar-jakub) [reply Project Management/Schedule/Zowe PI & Sprint Cadence.md:117]: Yes, that's the last one, nothing more after this one.

**Review by t1m0thyj — APPROVED** [review]

**Pull request merged**